### PR TITLE
updated phq9 component with orm data update and retrieve

### PR DESCRIPTION
**Pull request by tanmoyjana-grmtech**
updated phq9 component with orm data update and retrieve
phq9-edit-layer

phq9-view-layer

### CODE REVIEW

**Review by rajgrmtech — APPROVED** [review]

File: webclient/components/pt-info/single/1time-1row-mField/phq9/db/client-side/static-data/search-phrases-ct.vue [hunk 1]
```diff
@@ -1,11 +1,14 @@
+<!-- To understand how this works read components/core/search-phrases/load-search-phrases-and-handle-selection.vue -->
 <template>
-  <div></div>
+  <div><tblMasterListOfPhq9Ct /></div>
```
Comment (vikaskedia) [inline webclient/components/pt-info/single/1time-1row-mField/phq9/db/client-side/static-data/search-phrases-ct.vue:3]: Why is this needed?

**Review by vikaskedia — CHANGES_REQUESTED** [review]

File: webclient/components/pt-info/single/1time-1row-mField/phq9/db/client-side/static-data/search-phrases-ct.vue (hunk 1 above)
Comment (tanmoyjana-grmtech) [reply webclient/components/pt-info/single/1time-1row-mField/phq9/db/client-side/static-data/search-phrases-ct.vue:3]: in vue to add child comonent, needs to bind the component name in the template file as a with < child comonent />

File: webclient/components/core/search-phrases/load-search-phrases-and-handle-selection.vue [hunk 2]
```diff
@@ -77,7 +77,7 @@ import weightSPhrases from '@/components/pt-info/single/1time-1row-mField/bm/sub
 
 // 1time-1row-mField/others
 import mseSPhrases from '@/components/pt-info/single/1time-1row-mField/mental-status-exam/db/client-side/static-data/search-phrases-ct'
-import phq9SPhrases from '@/components/pt-info/single/1time-1row-mField/phq9/static-data/search-phrases-ct'
+import phq9SPhrases from '@/components/pt-info/single/1time-1row-mField/phq9/db/client-side/static-data/search-phrases-ct'
```
Comment (vikaskedia) [inline webclient/components/core/search-phrases/load-search-phrases-and-handle-selection.vue:80]: add file extension

**Review by vikaskedia — CHANGES_REQUESTED** [review]
